subsys/net/l2/openthread/openthread.c

```diff
@@ -279,6 +297,24 @@ enum net_verdict ieee802154_radio_handle_ack(struct net_if *iface,
 	return NET_CONTINUE;
 }
 
+void openthread_joiner_autostart(struct net_if *iface)
+{
+	struct openthread_context *ot_context = net_if_l2_data(iface);
+	otError error;
+
+	if (!otDatasetIsCommissioned(ot_context->instance)) {
+		error = otJoinerStart(ot_context->instance, CONFIG_OPENTHREAD_JOINER_PSKD, NULL,
```

Comment:
Can you reformat these three lines, they seem to be over 80 char limit?

Reply:
Thanks! I fixed it.